tools/lorisform_parser.php

```diff
@@ -54,7 +54,7 @@
     echo "Requiring file...\n";
     include_once $file;
     echo "Instantiating new object...\n";
-    $obj =new $className(new Module("", ""), "", "", "", "");
+    $obj =new $className(new NullModule("", ""), "", "", "", "");
```

Comment:
```suggestion
    $obj =new $className(new NullModule(), "", "", "", "");
```

This shouldn't be passing Travis, it has the wrong number of arguments for the constructor for NullModule..

Reply:
So there is no testcase for this or no rule? Strange.

Reply:
> So there is no testcase for this or no rule? Strange.

@johnsaigle maybe you can check the rules? The tools directory is not verified in the rules. I checked, there will be a lot errors using the current rules.

Reply:
Yeah Phan doesn't check the tools directory. It would be good to enable this. We should probably get #5766 merged first